src/components/modes/teacher/Settings.js

```diff
@@ -116,15 +117,22 @@ class Settings extends Component {
   };
 
   handleChangeInitialTimeValue = ({ target: { value } }) => {
-    this.setState({
-      initialTimeValue: Number(value),
-    });
+    if (Number(value) < 0) {
```

Comment:
Does this work correctly? If they start typing `-`, what shows?

Reply:
Maybe reset it to 0 if it is under 0?

Reply:
Error State - (It currently do not allow negative numbers)
![Screenshot 2020-04-23 at 13 42 42](https://user-images.githubusercontent.com/18397749/80105105-6416b180-8578-11ea-8982-183b2ca7146f.png)

Default State
![Screenshot 2020-04-23 at 13 42 03](https://user-images.githubusercontent.com/18397749/80105133-6d078300-8578-11ea-917b-7bcd872a5ad6.png)